includes/rules/img_alt_redundant.php

```diff
@@ -8,6 +8,8 @@
 /**
  * IMG ALT Redundant
  *
+ * @modified 1.11.0 Updated various conditions to use `empty` checks instead of checking for empty strings, since SimpleHTMLDom returns `false` for non-exist` attributes.
```

Comment:
```suggestion
 * @modified 1.11.0 Updated various conditions to use `empty` checks instead of checking for empty strings since Simple HTML Dom returns `false` for non-existent attributes.
```